tsconfig.json

```diff
@@ -2,23 +2,15 @@
     "compilerOptions": {
         "outDir": "./dist",
         "pretty": true,
-        "target": "es2015",
-        "lib": ["es2015"],
+        "target": "es2019",
+        "lib": ["es2019"],
         "module": "commonjs",
+        "allowJs": false,
         "sourceMap": true,
         "declaration": true,
         "importHelpers": true,
         "strict": true,
         "noImplicitAny": false
     },
-    "include": [
-        "lib",
-        "test"
-    ],
-    "files": [
-        "node_modules/typescript/lib/lib.es6.d.ts"
-    ],
-    "exclude": [
-        "node_modules"
-    ]
```

Comment:
is this no longer necessary?

Reply:
It's a good question, the only thing i know is that all other projects don't have it